File: modele_classes_csharp/Service.cs
using System.Collections.Generic;

namespace Moov.Facturation.Catalogue
{
    /// <summary>
    /// Service optionnel
    /// </summary>
    public class Service
    {
        // Attributs
        public string Id { get; set; }
        public string Nom { get; set; }
        public string Code { get; set; } // unique
        public TypeService TypeService { get; set; }
        
        // Relations
        public List<TarifService> Tarifs { get; set; }
        public List<Cycle> Cycles { get; set; }
        
        // Constructeur
        public Service()
        {
            Tarifs = new List<TarifService>();
            Cycles = new List<Cycle>();
        }
    }
}

File: modele_classes_java_simple/Service.java
import java.util.List;

/**
 * Service optionnel
 */
public class Service {
    
    // Attributs
    private String id;
    private String nom;
    private String code;
    private TypeService typeService;
    
    // Relations
    private List<TarifService> tarifs;
    private List<Cycle> cycles;
}
